README.md

```diff
@@ -1 +1,49 @@
-# fuel-v2-contracts
\ No newline at end of file
+# Fuel Solidity Contracts
+
+<!-- Disable markdownlint for long lines. -->
+<!-- markdownlint-disable-file MD013 -->
+
+![ci](https://github.com/fuellabs/fuel-v2-contracts/workflows/Node.js%20Tests%20and%20Coverage/badge.svg?branch=master)
+[![codecov](https://codecov.io/gh/fuellabs/fuel-v2-contracts/branch/master/graph/badge.svg?token=FVXeaaBA3d)](https://codecov.io/gh/fuellabs/fuel-v2-contracts)
+
+The Fuel Solidity smart contract architecture.
```

Comment:
```suggestion
The Fuel-Solidity smart contract architecture.
```

Reply:
In this case, it's not `fuel-solidity` the tool but really the Solidity implementation of Fuel stuff. E.g. Bitcoin Core instead of Bitcoin-Core https://bitcoin.org/en/bitcoin-core/.